enterprise/frontend/src/embedding-sdk/store/reducer.ts

```diff
@@ -32,17 +36,23 @@ const GET_OR_REFRESH_SESSION = "sdk/token/GET_OR_REFRESH_SESSION";
 
 export const getOrRefreshSession = createAsyncThunk(
   GET_OR_REFRESH_SESSION,
-  async (url: string, { dispatch, getState }) => {
+  async (
+    instanceUrl: MetabaseAuthConfig["metabaseInstanceUrl"],
+    { dispatch, getState },
+  ) => {
+    // necessary to ensure that we don't use a popup every time the user
+    // refreshes the page
+    const storedAuthToken = authTokenStorage.get();
```

Comment:
this function is located in `auth/saml-token-storage` I'd name this `samlTokenStorage` or something similar to make sure it's clear this isn't for JWT too.